src/components/ConfirmModal.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import {
+    TouchableOpacity, Text, View,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import Header from './Header';
+import Modal from './Modal';
+import styles from '../styles/styles';
+import CONST from '../CONST';
+import withWindowDimensions, {windowDimensionsPropTypes} from './withWindowDimensions';
+
+const propTypes = {
+    /** Title of the modal */
+    title: PropTypes.string.isRequired,
+
+    /** A callback to call when the form has been submitted */
+    onConfirm: PropTypes.func.isRequired,
+
+    /** A callback to call when the form has been closed */
+    onCancel: PropTypes.func.isRequired,
+
+    /** Modal visibility */
+    isVisible: PropTypes.bool.isRequired,
+
+    /** Confirm button text */
+    confirmText: PropTypes.string,
+
+    /** Cancel button text */
+    cancelText: PropTypes.string,
+
+    /** Modal content text */
+    prompt: PropTypes.string,
+
+    ...windowDimensionsPropTypes,
+};
+
+const defaultProps = {
+    confirmText: 'Yes',
+    cancelText: 'No',
+    prompt: '',
+};
+
+const ConfirmModal = props => (
+    <Modal
+        onSubmit={props.onConfirm}
+        onClose={props.onCancel}
+        isVisible={props.isVisible}
+        type={props.isSmallScreenWidth
+            ? CONST.MODAL.MODAL_TYPE.BOTTOM_DOCKED
+            : CONST.MODAL.MODAL_TYPE.CONFIRM}
+    >
+        <View style={styles.m5}>
+            <View style={styles.flexRow}>
+                <Header title={props.title} />
+            </View>
+
+            <Text style={[styles.textLabel, styles.mt4]}>
+                {props.prompt}
+            </Text>
+
+            <TouchableOpacity
+                style={[styles.button, styles.buttonSuccess, styles.mt4]}
+                onPress={props.onConfirm}
+            >
+                <Text
+                    style={[
+                        styles.buttonText,
+                        styles.buttonSuccessText,
+                        styles.buttonConfirmText,
```

Comment:
What does `buttonConfirmText` do versus `buttonSuccessText`? I'm not entirely sure why `buttonConfirmText` exists, it looks like it just adds padding to the left and the right but I actually don't think we need it. I would suggest we remove this.

Reply:
Yes. `styles.buttonConfirmText` is for add horizontal padding. Seems we don't need it here. I'll remove it.

Reply:
Awesome, thank you! Changes look good, you'll just need to update the screenshots again.

Reply:
Thanks! Screenshots updated.